recipes/sksparse/conda_build_config.yaml

```diff
@@ -0,0 +1,13 @@
+python:
+  - 3.6
+  - 3.7
+  - 3.8
+  - 3.9
+numpy:
+  - 1.16.5
+  - 1.16.5
+  - 1.17.3
+  - 1.19.2
+zip_keys:
+  - python
+  - numpy
```

Comment:
Why do we need this complex build matrix?

Reply:
Can't we build everything against numpy 1.17?

Reply:
I don't think so. python 3.9 isn't supported by 1.17 and the C-extensions in this package use numpy and the numpy ABI is only forward compatible so the binaries have to be build with the earliest supported version of numpy.

Reply:
Our default pinnings have all of these support ranges in them. We are adding python 3.9 via a migrator and so that build should be removed here. Please remove python 3.9 and this file of keys. 

Further, as noted below, it is not properly formatted to match our build strings in any case.

Also I noticed it is missing a zip key that will be needed for rendering.

Reply:
OK, i'm just deleting everything and we'll see what happens

Reply:
Thank you! We can add things back in the feedstock if needed and I am happy to help do that there.